res/config.json

```diff
@@ -3,18 +3,21 @@
     {
       "42": {
         "trunks": [
-          {
-            "42": {
-              "stations": [
-                {
-                  "endpoint": "SIP/100"
-                },
-                {
-                  "endpoint": "SIP/96@cwolfe"
-                }
-              ]
-            }
-          }
+          "42-A"
+        ],
+       "stations": [
+         "SIP/bobmarley",
+         "SIP/kingcrimson"
+        ]
+      }
+    },
+    {
+      "420": {
+        "trunks": [
+          "42-A"
+        ],
+       "stations": [
```

Comment:
I think it's your tab size is 3 rather than 4. The asterisk guidelines state that the tab size should be 4: https://wiki.asterisk.org/wiki/display/AST/Coding+Guidelines#CodingGuidelines-Codeformatting